docs/apis/avalanchego/apis/p-chain.md

```diff
@@ -1439,18 +1467,18 @@ Get the amount of nAVAX staked by a set of addresses. The amount returned does n
 #### **Signature**
 
 ```sh
-platform.getStake(
-    {addresses: []string}
-) ->
+platform.getStake({
+    addresses: []string
+}) ->
 {
-    staked: int,
+    stakeds: string -> int,
     stakedOutputs:  []string,
     encoding: string
 }
 ```
 
 - `addresses` are the addresses to get information about.
-- `staked` is the amount of nAVAX staked by addresses provided.
+- `stakeds` is a map from assetID to the amount staked by addresses provided.
```

Comment:
```suggestion
- `stakeds` is a map from assetID to the amount staked by addresses provided. The amount is denominated in 10^-9 of native token, e.g., nAVAX on the Primary Network. 
```

Reply:
Same as this comment https://github.com/ava-labs/avalanche-docs/pull/1053/files#r1007427585